backend/uclapi/.env.example

```diff
@@ -53,6 +54,7 @@ DB_CACHE_USERNAME=
 DB_CACHE_PASSWORD=
 DB_CACHE_HOST=
 DB_CACHE_PORT=5432
+DB_CACHE_POOL_SIZE=20
```

Comment:
This does not seem to be used anywhere, but I assume it's meant for [the update gencache script](https://github.com/uclapi/uclapi/blob/2da88489a11e6d1896a5baee69205da951c49cf7/backend/uclapi/timetable/management/commands/update_gencache.py#L333)? 
Same for DB_UCLAPI_POOL_SIZE and its respective file?

Reply:
This accurately represents the pools size we set at line 149 in settings.py, with 15 pool size and 5 for overflow, we planned I think on using the environment variable to set it, or we used to before we changed pooling mechanism maybe? Can safely be removed now at least I believe.

It does not reference the pool size in gencache.